internal/dao/sqlx/account_dao.go

```diff
@@ -91,8 +109,11 @@ func (di *accountDaoSqlx) GetOrCreateByIdentity(ctx context.Context, orgId strin
 	} else {
 		return nil, err
 	}
-	// TODO create HMSPROV-135
-	return nil, err
+	acc = &models.Account{OrgID: orgId, AccountNumber: ptr.String(accountNumber)}
```

Comment:
Is this reachable? I feel like this method is going to end after the first if-else statement.

Reply:
Okay, nevermind, it is reachable. But it might not check multiple results for two diffferent accounts - one matching the orgId, the other one matching account number, right?

Reply:
Yeah, that case is completely ignored in this implementation.
Although I feel like we should not reconcile on it, rather make sure it can never happen (by making the orgId actually unique in database)

Reply:
Also that brings up the thought of arranging this code in a way it would be clear it continues after thos if/else statements, I'll try to [keep line of sight](https://github.com/RHEnVision/provisioning-backend/pull/90#discussion_r923312186) :)

Reply:
I've improved the line of sight to make it bit more readable